### ISSUE TITLE
Send_config and support for multiline config commands 

### ISSUE THREAD

**Issue by MichalObs97** [ktbyers/nornir_netmiko#46]
Hello, 

I found a problem that I am trying to solve today. On Cisco Catalyst 9800 Wireless controller, when uploading a certificate to the controller, this interaction is happening in the config menu, so I am using send_config command. However, when I issue a command _crypto pki import...._ , next line controllers answers with: _Address or name of remote host..._ and does not show prompt again. 

And that is where I am getting stucked. On Send_command I can usu expect string, but I cannot find any solution for this in the send_config.

any help with multi line commands use in send_config? 
thanks

**Comment by ktbyers** [comment]
@MichalObs97 You probably need to use `send_command()`

Roughly:
```
conn.config_mode()
output = conn.send_command("command", expect_string=r"pattern")
output += conn.send_command("command2", expect_string=r"#")
conn.exit_config_mode()
```

**Comment by MichalObs97** [comment]
@ktbyers will `send_command()` let me use configure mode and find a prompt after?

**Comment by ktbyers** [comment]
@MichalObs97 Yes, `send_command()` let's you use the `expect_string` argument to search for patterns in the output.

Was that what you were asking?

**Comment by ktbyers** [comment]
Another solution would be the netmiko_multiline task plugin:

https://github.com/ktbyers/nornir_netmiko/blob/develop/nornir_netmiko/tasks/netmiko_multiline.py
